src/utils/paths.ts

```diff
@@ -3,5 +3,15 @@ export const APP_ROOT = '/finn-kandidat';
 export enum AppRoute {
     Oversikt = '/',
     Registrering = '/registrering',
+    SeKandidat = '/:fnr',
+    EndreKandidat = '/:fnr/endre',
     FørDuBegynner = '/for-du-begynner',
 }
+
+export interface MatchProps {
+    fnr: string;
+}
+
+export const getRouteWithFnr = (route: AppRoute, fnr: string) => {
```

Comment:
Finnes det et fornuftig norsk navn for denne metoden?

Reply:
`hentRoute` med optional `fødselsnummer`?